features/step_definitions/editionable_worldwide_organisation_steps.rb

```diff
@@ -73,9 +78,14 @@
   click_link "Reorder"
 end
 
+When(/^I visit the "([^"]*)" tab/) do |tab|
+  visit admin_worldwide_organisation_worldwide_offices_path(EditionableWorldwideOrganisation.last)
+  click_link tab
+end
+
 When(/^I visit the Offices tab/) do
```

Comment:
This step has been replaced by the above `I visit the "([^"]*)" tab` step, so should be removed.

Reply:
👍